app/src/main/java/com/example/myapplication/FirstFragment.java

```diff
@@ -49,79 +55,77 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
                 NavHostFragment.findNavController(FirstFragment.this)
                         .navigate(R.id.action_FirstFragment_to_ThirdFragment);
             } else {
-                if (!teamNumberCheck){
+                if (!teamNumberCheck) {
                     Snackbar.make(view, "Invalid team number", 600).show();
                 }
                 if (!matchNumCheck) {
                     Snackbar.make(view, "Invalid match number", 600).show();
                 }
-
             }
         });
+
         binding.back.setOnClickListener(v -> NavHostFragment.findNavController(FirstFragment.this)
                 .navigate(R.id.action_FirstFragment_to_HomePage));
+
         ObjectAnimator animation = ObjectAnimator.ofFloat(binding.pop, "rotation", 0f, 90f, 180f, 270f, 360f, 90f, 180f, 270f, 360f, 90f, 180f, 270f, 360f);
         animation.setDuration(1000);
         binding.pop.setOnClickListener(view1 -> {
             animation.start();
             UIHelpers.darkMode = !UIHelpers.darkMode;
             UIHelpers.lightDark(v, UIHelpers.darkMode);
         });
+
         DisplayMetrics dm = new DisplayMetrics();
         ((Activity) requireContext()).getWindowManager().getDefaultDisplay().getMetrics(dm);
         float width = dm.widthPixels;
         float height = dm.heightPixels;
+
         binding.title.setTranslationX(height * 0.072f);
-        binding.title.setTranslationX(width * 0.146f);
+        binding.title.setTranslationY(width * 0.146f);
```

Comment:
The `Y` translation should be relative to the height, not the width

Reply:
done